extensions/positron-r/src/extension.ts

```diff
@@ -43,4 +44,76 @@ export function activate(context: vscode.ExtensionContext) {
 			refreshTestExplorer(context);
 		}
 	});
+
+	// Listen for terminal output
+	const disposable = vscode.window.onDidWriteTerminalData(event => {
+		processTerminalOutput(event.data);
+	});
+	context.subscriptions.push(disposable);
```

Comment:
Will go away. Purely a development convenience, when convincing myself that we could see OSC 8 hyperlinks.